tests/test_shadow_fork.py

```
@@ -72,19 +72,29 @@ def test_create_fails_if_already_active(self, tmp_path):
+                # HEAD is not checked via _run_git, it just returns "HEAD"
+                (True, "", ""),  # status --porcelain (no uncommitted changes)
+                (True, "", ""),  # checkout -b shadow_branch HEAD
             ]
 
-            with pytest.raises(RuntimeError, match="does not exist"):
-                fork.create(base_branch="nonexistent")
+            # Create hooks directory for the test
+            (tmp_path / ".git" / "hooks").mkdir(parents=True)
+
+            fork.create(base_branch="nonexistent")
+            assert fork.base_branch == "HEAD"""
```

Ruff (0.15.17)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)